site2/docs/client-libraries-go.md

```diff
@@ -302,6 +302,91 @@ fmt.Println(msg.Payload())
 canc()
 ```
 
+#### How to use Prometheus metrics in producer
```

Comment:
Thanks @imryao work for this, good patch.

This title looks inconsistent with the content, what about `How to use Prometheus metrics in Go SDK`? Or It is best that we can separate the code examples of producer and consumer.

Reply:
@wolfstudy @Anonymitaet Thanks for your suggestions! I'll separate the code examples and add some description.

Reply:
I've separated the code examples and added some description. Looking forward to your further advice!